pkg/plugins/datasource/wrapper/datasource_plugin_wrapper_v2.go

```diff
@@ -76,7 +76,14 @@ func (tw *DatasourcePluginWrapperV2) Query(ctx context.Context, ds *models.DataS
 		})
 	}
 
-	pbRes, err := tw.DataPlugin.QueryData(ctx, pbQuery)
+	var pbRes *pluginv2.QueryDataResponse
+	err = backendplugin.InstrumentPluginRequest(ds.Type, "dataquery", func() error {
+		var err error
+		pbRes, err = tw.DataPlugin.QueryData(ctx, pbQuery)
+
+		return err
+	})
+
```

Comment:
Nit, but do you mind removing this blank line? I think it reads much clearer if the error check is colocated with the originating call.